File: 01_Basico/02 - Controle_Selecao/Resposta_em_Kotlin/questao05.kt
// Em kotlin: Utilizamos a função sorted, e depois a reverse para deixar em ordem decrescente e to para criar uma copia da lista, sem alterar a lista original
fun main(){
val numero1 = 50
val numero2 = 80
val numero3 = 60
val teste = mutableListOf<Int>(numero1, numero2, numero3)
var numeros = teste.toMutableList()
println(numeros.sorted().reversed())
print(teste)
}

// Outra forma com Controle de seleção:
fun main(){
   val nota1 = 10
   val nota2 = 4
   val nota3 = 2
   var maior = nota1
   var meio =  nota1
   var menor = nota1
    
  if (maior < nota2){
      maior = nota2
  }else{
      meio = nota2
      menor = nota2
  }
  if (maior < nota3){
      meio = maior
      maior = nota3
  }
  if (maior > nota3 && meio < nota3){
      meio = nota3
  }
  if (meio > nota3){
      menor = nota3
  }
  println("A maior nota é $maior, a nota do meio é $meio, a menor nota é $menor")   
}

File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao05.py
numero_1 = int(input("Digite um número: "))
numero_2 = int(input("Digite um número: "))
soma = 0
maior = numero_1 if (numero_1 > numero_2) else numero_2
menor = numero_1 if (numero_1 < numero_2) else numero_2

for i in range(menor,maior+1):
    soma += i
    print(i)

print(f"A soma dos número é {soma}")




